keps/sig-ui/5008-kep-template/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Kubernetes Desktop
+kep-number: 5008
+authors:
+  - "@joaquimrocha"
+owning-sig: sig-ui
+participating-sigs:
+status: provisional
+creation-date: 2024-12-19
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
no reviewers or approvers?

Reply:
I was not sure what to add here initially. Then I saw the bot was kind of guiding this so I assumed this would be added later. I will create a new PR to add these.

Reply:
This should not be merged without approval from leads of a sponsoring SIG, I think it only merged because @johnbelamaric approved it but that was for Production Readiness Review only. It needs both the approval of the involved SIG(s) and production readiness review (which is cross-sig and aimed and ensuring stability and the feature graduation process)

Reply:
Ah sorry, I thought SIG-UI had approved. I should have waited but since it was provisional I went ahead.

Reply:
@BenTheElder , the sponsoring SIG ( @floreks or @maciaszczykm ) members did give their LGTM. I thought that counted as approval. Is there something else needed from them?

Reply:
Ah, preferably this should be done via /approve which is recorded in the robot approval message, and also listed reviewers / approvers in the KEP metadata (this can be reflected elsewhere for KEP tracking). This prevents it being lost deep in the thread 🙃

Reply:
https://github.com/kubernetes/enhancements/pull/5009#issuecomment-2649160530 (Makes it look like John only, contributors are used to looking for the approval status in this comment which will be updated down the thread with the current status)

Reply:
I think sig-ui was a new directory, so there were no sig-ui OWNERS